packages/mds-web-sockets/server.ts

```diff
@@ -1,14 +1,19 @@
 import logger from '@mds-core/mds-logger'
 import { seconds, getEnvVar } from '@mds-core/mds-utils'
+import { Json } from '@mds-core/mds-types'
 import WebSocket from 'ws'
 import { setWsHeartbeat } from 'ws-heartbeat/server'
-import { Telemetry, VehicleEvent } from '@mds-core/mds-types'
 import { ApiServer, HttpServer } from '@mds-core/mds-api-server'
 import { initializeNatsSubscriber } from '@mds-core/mds-stream/nats/nats'
 import { Clients } from './clients'
-import { ENTITY_TYPE } from './types'
-
-export const WebSocketServer = () => {
+import { EntityTypes } from './types'
+
+/**
+ * Web Socket Server that pas
+ * @param entityTypes - entities to pass on to clients
+ */
```

Comment:
I'm down with having some JSDoc in here, but looks like there's a typo

Reply:
updated